### PR TITLE
Add thumbor Docker container and have it run in ECS

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What is this PR trying to achieve?
As we'll be losing the Wordpress image resizer, and Prismic doesn't come with one, e need to roll our own (or, in this case, just host a well tested solution).

We'll need to add some perf metrics in here, and cache.

Next will be adding this to the build.

There's also a need to maybe start tidying the Terraform - but it works and is clear enough for now.

## What does it look like?
🎨

### CODE REVIEW

File: infra/terraform/container-definitions-thumbor.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "name": "thumbor",
+    "image": "wellcome/wellcomecollection-thumbor:test",
+    "cpu": 512,
+    "memory": 1000,
```
Comment (alexwlchan) [inline infra/terraform/container-definitions-thumbor.json]: I’d have to check, but isn’t the max memory on an instance a multiple of 1024? ECS surely won’t be able schedule a second container in that remaining 24 MiB, so it’s just wasted.

File: infra/terraform/container-definitions.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
   {
     "name": "nginx-proxy",
     "image": "wellcome/wellcomecollection-nginx-proxy:${container_tag}",
-    "memory": 232,
-    "cpu": 128,
+    "cpu": 256,
+    "memory": 500,
```
Comment (alexwlchan) [inline infra/terraform/container-definitions.json]: Ditto above: maybe 512?

File: infra/terraform/templates/s3.tf [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  bucket = "${var.alb_log_bucket}"
+}
+
+data "aws_iam_policy_document" "alb_log_bucket_policy_document" {
+  statement {
+    sid = ""
+    effect = "Allow"
+    actions = ["s3:PutObject"]
+    resources = [
+        "arn:aws:s3:::${var.alb_log_bucket}/${var.alb_log_prefix}/AWSLogs/*"
```
Comment (alexwlchan) [inline infra/terraform/templates/s3.tf]: Maybe use `${var.alb_log_bucket.arn}` for the first part?

File: infra/terraform/container-definitions-thumbor.json (hunk 1 above)
Comment (jamesgorrie) [inline infra/terraform/container-definitions-thumbor.json]: a `t2.small` has 2GB, and after running docker, has 2003MB.

Comment (jamesgorrie) [inline infra/terraform/container-definitions-thumbor.json]: [I have the count of tasks set to 1](https://github.com/wellcometrust/wellcomecollection.org/pull/871/files#diff-2da4d8e5604afe8134b645552b93f62dR61) for testing - so would need to either up the instances, or lower the momory.

I just want to get a gist for how much memory etc image processing has.

**Pull request merged**